de_web_plugin.cpp

```diff
@@ -1816,9 +1816,9 @@ void DeRestPluginPrivate::updateEtag(QString &etag)
 {
     QTime time = QTime::currentTime();
 #if QT_VERSION < 0x050000
-    etag = QString(QCryptographicHash::hash(time.toString().toAscii(), QCryptographicHash::Md5).toHex());
+    etag = QString(QCryptographicHash::hash(time.toString("hh:mm:ss.zzz").toAscii(), QCryptographicHash::Md5).toHex());
```

Comment:
why do you use *only* time for etag hash, why not ad date too ?
and why not add measurement values and/or other stuff too ?
the md5 hash is much longer than the raw time string (12 vs. 32 chars), I don't get it what here a md5 hash can be good for ?!?